README.md

```diff
@@ -30,49 +29,518 @@ and point your browser to http://localhost:8080.
+
+
+#### Attributes, which are scala keywords
+There are some attributes and styles which are reserved scala keywords. You can use them with backticks:
+
+```scala
+div(`class` := "item", "My Item")
+// <div class="item">My Item</div>
+
+label(`for` := "inputid")
+// <label for="inputid" />
+```
+
+There is also a shortcut for the class atrribute:
+
+```scala
+div(cls := "myclass")
+// <div class="myclass"></div>
+```
+
+Source Code: [OutwatchAttributes.scala](src/main/scala/outwatch/dom/OutwatchAttributes.scala#L65)
+
+
+#### Overriding attributes
+Attributes and styles with the same name will be overwritten. Last wins.
+
+```scala
+div(color := "blue", color := "green")
+// <div style="color: green"></div>
+```
+
+Source Code: [DomUtils.scala](src/main/scala/outwatch/dom/helpers/DomUtils.scala#L8)
+
+#### Class accumulation
+Classes are not overwritten, they accumulate.
+
+```scala
+div(cls := "tiny", cls := "button")
+// <div class="tiny button"></div>
+```
+
+#### Custom attributes, styles and tags
+All the tags, attributes and styles available in outwatch come from [Scala Dom Types](https://github.com/raquo/scala-dom-types).
+If you want to use something not available in Scala Dom Types, you can use custom builders:
+
+```scala
+htmlTag("app")(style("user-select") := "none", attr("everything") := "possible")
+// <app style="user-select: none" everything="possible"></div>
+```
+
+You can also define the accumulation behavior of custom attributes:
+```scala
+div(
+  attr("everything").accum("-") := "is",
+  attr("everything").accum("-") := "possible",
+)
+// <div everything="is-possible"></div>
+```
+
+If you think there is something missing in Scala Dom Types, please open a PR or Issue. Usually it's just one line of code.
+
+Source Code: [DomTypes.scala](src/main/scala/outwatch/dom/DomTypes.scala)
+
+
+#### Data attributes
+Data attributes make use of [`scala.Dynamic`](https://www.scala-lang.org/api/current/scala/Dynamic.html), so you can write things like:
+
+```scala
+div(data.payload := "17")
+// <div data-payload="17"></div>
+```
+
+Source Code: [OutwatchAttributes.scala](src/main/scala/outwatch/dom/OutwatchAttributes.scala#L70), [Builder.scala](src/main/scala/outwatch/dom/helpers/Builder.scala#L39)
+
+
+#### SVG
+SVG tags and attributes are available via an extra import. Namespacing is automatically handled for you.
+
+```scala
+val graphic = {
+ import svg._
+ svg(
+   viewBox := "0 0 10 10",
+   g(
+     transform := "matrix(.096584 0 0 .096584 -.0071925 -18.66)",
+     path(d := "M10 10 C 20 20, 40 20, 50 10", fill := "mistyrose")
+   )
+ )
+}
+
+// <svg viewBox="0 0 10 10"><g transform="matrix(.096584 0 0 .096584 -.0071925 -18.66)"><path d="M10 10 C 20 20, 40 20, 50 10" fill="mistyrose"></path></g></svg>
+```
+
+**TODO**: Make an svg which actually shows something.
+
+Custom SVG attributes and tags are possible like this:
+
+```scala
+TODO: svgTag? svgAttr?
```

Comment:
Should we remove this if it's not possible yet? :)

Reply:
Ah, yes, I will move this into the documentation issue.